apps/next-evans/pages/friend/[friend_name].tsx

```diff
@@ -0,0 +1,68 @@
+import { PrismaClient } from '@prisma/client';
+import type { GetStaticPaths, GetStaticProps } from 'next';
+import FeaturedQuoteBlock from '@/components/pages/friend/FeaturedQuoteBlock';
+import FriendBlock from '@/components/pages/friend/FriendBlock';
+
+export const getStaticPaths: GetStaticPaths = async () => {
+  const prisma = new PrismaClient();
+  const allFriends = await prisma.friends.findMany();
+  const paths = allFriends.map((friend) => ({
+    params: {
+      friend_name: friend.slug,
+    },
+  }));
+
+  return {
+    paths,
+    fallback: false,
+  };
+};
+
+export const getStaticProps: GetStaticProps<Props> = async (context) => {
+  const prisma = new PrismaClient();
+  if (typeof context.params?.friend_name !== `string`) {
+    return {
+      notFound: true,
+    };
+  }
+  const friend = await prisma.friends.findFirst({
+    where: { slug: context.params.friend_name },
+  });
+  if (!friend) {
+    return {
+      notFound: true,
+    };
+  }
+  const friendQuote = await prisma.friend_quotes.findFirst({
```

Comment:
ok, i missed this the first time, but since you also bit off grabbing the friend quote, i'd like you to do two things different here...

first, can you try to _join_ and grab these in a single query?  i think build times for this app might get long, since we're going to be querying lots of friends, documents, editions, etc., so wherever we spot an opportunity to decrease queries (and shorten the code) we should probably take it. i'm pretty sure prisma supports joining pretty precisely.

second, instead of grabbing just the _first_ quote, grab all of them.  there is a component called `TestimonialsBlock` that we put all of the rest of the quotes in (if any). check out the page for Daniel Wheeler on the existing site for an example of one of these. On the real site it's at the bottom of the page, but for now, just render it under the featured quote block.